akka-samples/akka-docs-java-lambda/src/test/java/docs/stream/ActorPublisherDocTest.java

```diff
@@ -0,0 +1,153 @@
+/*
+ * Copyright (C) 2015 Typesafe Inc. <http://www.typesafe.com>
+ */
+package docs.stream;
+
+import akka.actor.ActorRef;
+import akka.actor.ActorSystem;
+import akka.actor.Props;
+import akka.japi.pf.ReceiveBuilder;
+import akka.stream.ActorFlowMaterializer;
+import akka.stream.FlowMaterializer;
+import akka.stream.actor.AbstractActorPublisher;
+import akka.stream.actor.ActorPublisherMessage;
+import akka.stream.javadsl.KeyedSource;
+import akka.stream.javadsl.MaterializedMap;
+import akka.stream.javadsl.Sink;
+import akka.stream.javadsl.Source;
+import akka.testkit.JavaTestKit;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ActorPublisherDocTest {
+
+  static ActorSystem system;
+  
+
+  @BeforeClass
+  public static void setup() {
+    system = ActorSystem.create("ActorPublisherDocTest");
+  }
+
+  @AfterClass
+  public static void tearDown() {
+    JavaTestKit.shutdownActorSystem(system);
+    system = null;
+  }
+  
+  final FlowMaterializer mat = ActorFlowMaterializer.create(system);
+  
+    //#job-manager
+    public static class JobManagerProtocol {
+      final public static class Job {
+        public final String payload;
+
+        public Job(String payload) {
+          this.payload = payload;
+        }
+
+      }
+
+      public static class JobAcceptedMessage {}
+      public static final JobAcceptedMessage JobAccepted = new JobAcceptedMessage () { 
+        @Override
+        public String toString() { return "JobAccepted"; }
```

Comment:
doesn't it make more sense to have the toString in the `JobAcceptedMessage` class?